setup.cfg

```
@@ -59,6 +59,7 @@ install_requires =
     requests
     python-dateutil
     python-dotenv
+    univers == 30.11.0
```

Please never pin the version here. Only express what is your minimal required version. The pin goes in the requirements file.

```suggestion
    univers >= 30.11.0
```

---

Glad you caught that -- thanks and fixed.